src/iterators/mod.rs

```diff
@@ -1040,7 +1040,7 @@ impl<'a, A, D: Dimension> NdProducer for AxisIterMut<'a, A, D>
 /// See [`.axis_chunks_iter()`](../struct.ArrayBase.html#method.axis_chunks_iter) for more information.
 pub struct AxisChunksIter<'a, A: 'a, D> {
     iter: AxisIterCore<A, D>,
-    last_ptr: *mut A,
+    original_len: usize,
```

Comment:
Will you please add a comment describing what this is?

```rust
/// Length of iterator before starting. (It's necessary to preserve the length
/// because `self.iter.next_back()` changes `self.iter.len`.)
```

or

```rust
/// Number of chunks.
```